### PR TITLE
Feature/radical.saga

### PR DESCRIPTION
**Pull request by andre-merzky**
This has been in the works for a long time: it moves `saga` to `radical.saga`, and at the same time overhauls the broken configuration system.  Configuration management is now provided by RU, and is designed so that it also simplifies configuration handling on RP level.

The PR further fixes a couple of unit testing issues, so that at least the base and local tests should now pass.

### CODE REVIEW

**Comment by andre-merzky** [comment]
Merging this depends on radical-cybertools/radical.utils/pull/130.

**Comment by mturilli** [comment]
Should `'url'                : 'http://radical-cybertools.github.io/saga-python/',` in `setup.py` be changed? More generally, are we going to change the name of the repository to `radical.saga`?

**Comment by mturilli** [comment]
I pushed an update to README but I did not update the link to the repository and documentation waiting to know whether we are going to change the name of the repository.

**Comment by mturilli** [comment]
All the lines `__copyright__ = "Copyright 2012-2013, The SAGA Project"` should probably updated to 2018. We should find a way to do that automatically, assuming we want to keep the copyright note at all...

**Comment by andre-merzky** [comment]
See [here](https://github.com/radical-cybertools/radical.utils/pull/130) for the related `radical.utils` PR.

File: src/radical/saga/__init__.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+
+__author__    = "SAGA Development Team"
```
[the remark this answers is not in the document]
Comment (andre-merzky) [reply src/radical/saga/__init__.py]: This is fixed now.

**Pull request merged**